app/assets/v2/js/shared.js

```diff
@@ -1201,6 +1201,39 @@ function renderBountyRowsFromResults(results, renderForExplorer) {
   return html;
 }
 
+function saveAttestationData(result, cost_eth, to_address, type) {
```

Comment:
could we change it to variable functions `const saveAttestationData = (result, cost_eth, to_address, type) => { }`  ?